lua/weapons/gmod_tool/stools/wire_door.lua

```diff
@@ -55,18 +55,26 @@ if !ConVarExists("sbox_maxdoors") then CreateConVar("sbox_maxdoors", defaultlimi
 		entit:SetPlayer( ply )
 		ply:AddCount( "doors", entit )
 		ply:AddCleanup( "doors", entit )
-		
+
 		local index = ply:UniqueID()
 		Doors[ index ] 			= Doors[ index ] or {}
 		Doors[ index ][1] 	= Doors[ index ][1] or {}
 		table.insert( Doors[ index ][1], entit )
-		
-		
+
+		local door = entit.Entity:makedoor(ply,trace,ang,model,open,close,autoclose,closetime,class,hardware)
+
 		undo.Create("Door")
 		undo.AddEntity( entit )
+
+		if IsValid( door ) then
+			undo.AddEntity( door )
+			ply:AddCleanup( "doors", door )
+		end

```

Comment:
If it correctly made the door we add it to the undo (which should set CPPI ownershio) and adds it to the same cleanup list

Reply:
Good call. Both Nadmod and FPP check `cleanup.Add` - I'll use that here instead of `Player:AddCleanup`